corehq/apps/domain/templates/domain/admin/feature_flags.html

```diff
@@ -1,4 +1,4 @@
-{% extends "settings/base_template.html" %}
+{% extends "settings/base_template_b3.html" %}
```

Comment:
what i've been doing for dual templates is `settings/base_template.html` ->  `settings/bootstrap2/base_template.html` and then the b3 templates in `settings/bootstrap3/base_template.html` that way it's clear that, hey, this page is obv still using b2 when there is probably a b3 template available.